clients/go/grpc/sample.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
```

Comment:
@AkbaraliShaikh if this used for example, then lets move it to an example folder only or testdata

Reply:
yes, It is only for the test. will move to `testdata`

Reply:
updated